src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="link"
+        href="../src/index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="page-navigation">
+        <ul class="menu list">
+          <li class="menu-item">
+            <a
+              class="menu-link nav-link link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
```

Comment:
Keep your attributes correctly formatted as you did in lines 33-39. Each attribute should start on a new line with 2-space indentation.